src/components/app/MenuButtons.js

```diff
@@ -81,7 +81,7 @@ type ProfileSharingButtonProps = {|
   +buttonClassName: string,
   +shareLabel: string,
   +symbolicationStatus: string,
-  +okButtonClickEvent: () => void,
+  +okButtonClickEvent: () => Promise<void>,
```

Comment:
nit: because we don't really care about the return value here, you can use `mixed` (this is a `any` that doesn't disconnect Flow)
```suggestion
  +okButtonClickEvent: () => mixed,
```